backend/app/alembic/versions/5c4028ff26df_update_pop_udf.py

```diff
@@ -0,0 +1,89 @@
+"""update pop udf
+
+Revision ID: 5c4028ff26df
+Revises: ccb2a6b81a8b
+Create Date: 2024-10-07 20:30:15.230254
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from app.constants import SQL_DIR
+
+
+# revision identifiers, used by Alembic.
+revision: str = "5c4028ff26df"
+down_revision: Union[str, None] = "ccb2a6b81a8b"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint("districtr_map_uuid_unique", "districtrmap", ["uuid"])
+    op.create_unique_constraint(
+        "districtr_map_parent_child_edge_unique",
+        "parentchildedges",
+        ["districtr_map", "parent_path", "child_path"],
+    )
+    with open(f"{SQL_DIR}/total_pop_udf.sql") as f:
+        query = f.read()
+    op.execute(sa.text(query))
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(
+        "districtr_map_parent_child_edge_unique", "parentchildedges", type_="unique"
+    )
+    op.drop_constraint("districtr_map_uuid_unique", "districtrmap", type_="unique")
+    op.execute(
+        sa.text("""
```

Comment:
oh, this is nice, I was worried about our migrations. I like this better than either having to create new files for new versions or do something very convoluted to get prior versions of functions.